hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientManager.java

```diff
@@ -76,7 +81,8 @@ public XceiverClientManager(Configuration conf) {
         SCM_CONTAINER_CLIENT_MAX_SIZE_DEFAULT);
     long staleThresholdMs = conf.getTimeDuration(
         SCM_CONTAINER_CLIENT_STALE_THRESHOLD_KEY,
-        SCM_CONTAINER_CLIENT_STALE_THRESHOLD_DEFAULT, TimeUnit.MILLISECONDS);
+        //SCM_CONTAINER_CLIENT_STALE_THRESHOLD_DEFAULT, TimeUnit.MILLISECONDS);
```

Comment:
Is there a reason to hard code the stale threshold here?

Reply:
It's changed for unit test. Forget to remove it.